manifests/kiali-community/1.57.0/manifests/kiali.v1.57.0.clusterserviceversion.yaml

```diff
@@ -485,6 +485,7 @@ spec:
           - security.istio.io
           - extensions.istio.io
           - telemetry.istio.io
+          - gateway.networking.k8s.io
```

Comment:
Does this throw errors if the gateway APIs are not deployed into the cluster? Since the gateway API CRDs are not required and bundled with Istio, I wonder if we first need to check if those types exist before granting permissions for them or if kube doesn't care if you grant permissions for a type that doesn't exist in the cluster yet.

Reply:
No error on a clean cluster without Gateway API CRD, Kiali installs and runs fine.

Reply:
no - no APIs defined in roles like this are required to be deployed. This is why we can have one list that even contains openshift.io APIs (and it works on, say, minikube).